Angular/src/app/project-participants/project-participants.component.ts

```diff
@@ -14,7 +14,7 @@ import { RequestStatus } from '../request-status-enum';
 export class ProjectParticipantsComponent implements OnInit {
 
   projectParticipationRequest = new MatTableDataSource<ProjectParticipationRequest>();
-  showActionButtons: boolean = true;
```

Comment:
Why did you delete variable's type?

Reply:
type declaration was unnecessary. that action was performed by ts-lint